spec/checkPullRequestReviewSpec.js

```diff
@@ -77,6 +82,74 @@ describe('Pull Request Review Module', () => {
       };
     });
 
+    describe('When reviewer requests changes and LGTM label ' +
+     'was already added to the pull request.', ()=>{
+      beforeEach(async () => {
+        const label = {
+          id: 248679580,
+          node_id: 'MDU6TGFiZWwyNDg2Nzk1ODA=',
+          url: 'https://api.github.com/repos/oppia/oppia/labels/PR:%20LGTM',
+          name: 'PR: LGTM',
+          color: '009800',
+        };
+        // Set the payload action and label which will simulate removing
+        // the LGTM label.
+        payloadData.payload.label = label;
+        payloadData.payload.pull_request.requested_reviewers = [
+          { login: 'reviewer1' },
+          { login: 'reviewer2' },
+        ];
+        payloadData.payload.pull_request.assignees = [];
+        // Set project owner to be pr author.
+        payloadData.payload.pull_request.user.login = 'kevintab95';
+        reviewPayloadData.payload.pull_request.labels.push(label);
+        await robot.receive(reviewPayloadData);
+      });
+
+      it('should check type of review', () => {
+        expect(
+          pullRequestReviewModule.handlePullRequestReview
+        ).toHaveBeenCalled();
+      });
+
+      it('should wait for 3 minutes before performing any action', () => {
+        expect(utilityModule.sleep).toHaveBeenCalled();
+        expect(utilityModule.sleep).toHaveBeenCalledWith(
+          utilityModule.THREE_MINUTES);
+      });
+
+      it('Should comment on PR', () => {
+        expect(github.issues.createComment)
+          .toHaveBeenCalled();
+      });
+
+      it('Should Remove the LGTM Label', () => {
+        expect(github.issues.removeLabel).toHaveBeenCalled();
+      });
+
+      afterAll(() => {
+        const label = {
+          id: 248679580,
+          node_id: 'MDU6TGFiZWwyNDg2Nzk1ODA=',
+          url: 'https://api.github.com/repos/oppia/oppia/labels/PR:%20LGTM',
+          name: 'PR: LGTM',
+          color: '009800',
+        };
```

Comment:
This wouldn't work. 
To fix this, you need to store the labels before adding a new label and then setting the labels here to be what was stored initially.

See https://github.com/oppia/oppiabot/blob/a9c06dc340450a5d40cd3aa42bebaec421650b7c/spec/staleBuildChecksSpec.js#L360-L375

Reply:
Ok, Sir.  Thanks for the help. I will do it just now.